### DISCUSSION TITLE
Enhancing uBlock Origin with HaGeZi Filters and Threat Intelligence Feeds built-in

### DISCUSSION THREAD

**Discussion by ghost** [uBlockOrigin/uAssets#24108] [Ideas]
HaGeZi's filters are among the most curated and most stable filters available, featuring very low false positive rates. They often block trackers, scam, phishing, and malicious websites that uBlock Origin's built-in filters miss.

I recommend incorporating any HaGeZi filter, such as Normal, Pro, Pro Plus, or preferably Ultimate, into uBlock Origin's built-in filters. This addition would greatly enhance protection against trackers and potential malware/phishing threats. While it doesn't need to be enabled by default, having it as an option would be highly beneficial.

Furthermore, adding HaGeZi's Threat Intelligence Feed would significantly enhance user protection from all kinds of malicious attacks, as it includes over a million malicious URLs and is frequently updated with data from dozens of high-quality sources, as well as user contributions. Although incorporating the Full filter might be impractical due to its size, there are smaller, yet still powerful and useful, lower-tier filters that could be utilized effectively.

`https://github.com/hagezi/dns-blocklists`

**Comment by garry-ut99** [comment]
The request has been already declined for uBlock Origin Lite, hence most likely for uBlock Origin declined too:
- https://github.com/uBlockOrigin/uBOL-home/issues/120

<hr>

by the way from https://github.com/uBlockOrigin/uBlock-issues/blob/master/.github/ISSUE_TEMPLATE/bug_report.yml :
> New filter list requests are not permitted and will be declined and closed.

Though it's unclear what does "new" exactly mean:
- only creating new lists from scratch 
(example: https://github.com/uBlockOrigin/uBlock-issues/discussions/3262)
- OR adding already existing filter lists
- OR the both cases
